Add synthetic control as analysis method (#168)
david26694/cluster-experiments: README.md

The library offers the following classes:

* Regarding power analysis:

>     * `PowerAnalysis`: to run power analysis on a clustered/switchback design

Reviewer: ```suggestion
    * `PowerAnalysis`: to run power analysis on any experiment design, using simulation
```

    * `PowerAnalysisWithPreExperimentData`: to run power analysis on a clustered/switchback design, but adding pre-experiment df during split and perturbation (especially useful for Synthetic Control)

>     * `PowerAnalysis`: to run power analysis on any experiment design, using simulation

Reviewer: ```suggestion
```
